flink-connectors/flink-connector-pulsar/src/test/java/org/apache/flink/connector/pulsar/source/PulsarUnorderedSourceITCase.java

```diff
@@ -32,12 +32,14 @@
 import org.apache.flink.streaming.api.CheckpointingMode;
 
 import org.apache.pulsar.client.api.SubscriptionType;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Tag;
 
 /**
  * Unit test class for {@link PulsarSource}. Used for {@link SubscriptionType#Shared} subscription.
  */
 @Tag("org.apache.flink.testutils.junit.FailsOnJava11")
+@Disabled("FLINK-28352")
```

Comment:
Just asking: shouldn't it refer to FLINK-30351 where in theory it should be fixed?

Reply:
I guess, both are fine. I decided to go for the parent issue because it also includes the subtasks. But reiterating over it: You might be right because FLINK-30351 actually mentions all the subtasks that need to be resolved before removing the annotations again.